### PR TITLE
Always show bottom navigation

### PR DESCRIPTION
**Pull request by gediminasel**

### CODE REVIEW

File: app/src/main/java/com/nmakademija/nmaakademija/adapter/ArticlesAdapter.java [hunk 1]
```diff
@@ -2,21 +2,60 @@
...
 
 import com.bumptech.glide.Glide;
 import com.nmakademija.nmaakademija.ArticleActivity;
 import com.nmakademija.nmaakademija.R;
 import com.nmakademija.nmaakademija.entity.Article;
 
-import java.util.List;
+import java.util.ArrayList;
 
-public class ArticlesAdapter extends RecyclerView.Adapter<ArticlesAdapter.MyViewHolder> {
```
Comment (gediminasel) [inline app/src/main/java/com/nmakademija/nmaakademija/adapter/ArticlesAdapter.java:19]: Reikės ištrinti komentarą

**Comment by kuprelis** [comment]
RecyclerView efektyvesnis dalykas :-1:

File: app/src/main/java/com/nmakademija/nmaakademija/adapter/ArticlesAdapter.java [hunk 2]
```diff
@@ -2,21 +2,60 @@
...
+public class ArticlesAdapter extends ArrayAdapter<Article> {
```
Comment (vycius) [inline app/src/main/java/com/nmakademija/nmaakademija/adapter/ArticlesAdapter.java:20]: Why switched back from `ArrayAdapter` to `RecyclerView` ?

File: app/src/main/java/com/nmakademija/nmaakademija/fragment/NewsFragment.java [hunk 3]
```diff
@@ -42,8 +42,8 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
             public void onResponse(Call<List<Article>> call, Response<List<Article>> response) {
                 View v = getView();
                 if (v != null) {
-                    RecyclerView rv = (RecyclerView) getView().findViewById(R.id.recyclerView);
-                    rv.setAdapter(new ArticlesAdapter(response.body()));
+                    ListView listView = (ListView) getView().findViewById(R.id.recyclerView);
+                    listView.setAdapter(new ArticlesAdapter(getContext(), (ArrayList<Article>) response.body()));
```
Comment (vycius) [inline app/src/main/java/com/nmakademija/nmaakademija/fragment/NewsFragment.java:46]: Change `ArticlesAdapter` constructor to take `List<Article>` as an argument. If possible avoid such casts

**Pull request closed without merging**